js/client-battle-tooltips.js

```diff
@@ -1152,6 +1165,47 @@ var BattleTooltips = (function () {
 			basePower = Math.floor(basePower * abilityBoost);
 			basePowerComment = this.makePercentageChangeText(abilityBoost, ability);
 		}
+
+		var allyActive = pokemon.side.active;
+		var auraBoosted = '';
+		var auraBroken = false;
+		if (move.category !== 'Status' && allyActive.length > 1) {
+			for (var i = 0; i < allyActive.length; i++) {
+				var ally = allyActive[i];
+				if (!ally || ally.fainted) continue;
+				if (!auraBoosted && (ally.ability === 'Fairy Aura' && move.type === 'Fairy') ||
+						    (ally.ability === 'Dark Aura' && move.type === 'Dark')) {
+					auraBoosted = ally.ability;
+				}
+				if (ally.ability === 'Aura Break') auraBroken = true;
+				if (ally === pokemon) continue;
+				if (ally.ability === 'Battery' && move.category === 'Special') {
+					basePower = Math.floor(basePower * 1.3);
+					basePowerComment = this.makePercentageChangeText(1.3, 'Battery');
+				}
+			}
+		}
+		var foeActive = this.battle.yourSide.active;
+		if (!auraBoosted && move.category !== 'Status' && (move.type === 'Fairy' || move.type === 'Dark')) {
+			for (var i = 0; i < foeActive.length; i++) {
+				var foe = foeActive[i];
+				if (!foe || foe.fainted) continue;
+				if (!auraBoosted && (foe.ability === 'Fairy Aura' && move.type === 'Fairy') ||
+						    (foe.ability === 'Dark Aura' && move.type === 'Dark')) {
```

Comment:
This indentation is still dumb even if `eslint` doesn't complain about it. This is literally only designed for 8-space tabs, which we don't actually use.